cypress/support/step_definitions/dictionary/addBulkConceptsToDictionary/addBulkConceptsToDictionary.ts

```diff
@@ -0,0 +1,141 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import { 
+  After, 
+  Before, 
+  Given, 
+  Then, 
+  When } from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser } from '../../../utils';
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/concepts/`
+  );
+});
+
+Given('the user is on the "Add concepts in bulk from CIEL" page', () => {
+  cy.visit(
+    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+  
+  cy.url().should(
+    "contain",
+    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+Given('CIEL concept "1000" is already in the dictionary', () => {
+  cy.getConcept(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/1000`).should("exist");
+});
+
+When('the user clicks the "Add concepts" floating action button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+When('the user clicks the "Add concepts" MuiButton-label button', () => {
+  cy.findByText("Add concepts").click();
+});
+
+When('the user selects "Add bulk concepts"', () => {
+  cy.visit(
+    `/users/${getUser()}/collections/${getDictionaryId()}/add/?fromSource=CIEL/`
+  );
+});
+
+When(
+  /the user enters concept Id "(1000|1001|1002)"/, (conceptId) => {
+    cy.get("textarea").type(conceptId)
+  }
+); 
+
+When('the user re-enters concept Id "1000"', (conceptId) => {
+    cy.get("textarea").type(conceptId)
+  }
+); 
+
+When('the user navigates to the "Progress notifications" page', () => {
+  cy.visit(
+    `/actions/`
+  );
+});
+
+Then('the user selects "Import existing concept"', () => {
+   cy.findByText("Import existing concept").click();
+});
```

Comment:
Any action performed by the user should be a `When` step rather than a `Then` step.

* `Given` is for how things should be when the test starts
* `When` is for what actions the user takes (the actions we are testing)
* `Then` is for the steps to see if the action did what we expected

Another way to think of it is:

* If I am saying "the user is" this should be a `Given` step, e.g. "the user is on the home page"
* If I am saying "the user does something" this should be a `When` step, e.g. "the user clicks on the red button"
* If I am saying "the user should..." this should be a `Then` step, e.g. "the user is on the new page"